From 4ae828f2cbe4a49a032488f555d03495cb0aea3b Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Wed, 15 Aug 2018 10:11:54 +0800
Subject: [PATCH 4419/5242] MLK-19222 ARM64: dts: fsl-imx8mm-evk: enable
 USBOTG2

commit  70527b4f32e8e7a794f454c090d812c997178c66 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable USBOTG2 port, there are some limitations for USB port usage
from hardware engineer:

USBOTG2 port must be connected to Host or Charger at anytime as the
power supply.
USBOTG1 port can be connected to Host, device or charger only after
USBOTG2 port is connected.
This limitation is caused by SIP team's load switch, the debouce time
is too long, so when power supply switches from one port to another,
system might lose power and reboot.

Acked-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index f20f18f..6f94ac6 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -699,7 +699,7 @@
 		max-snk-mw = <15000>;
 		port-type = "drp";
 		default-role = "sink";
-		status = "disabled";
+		status = "okay";
 	};
 };
 
@@ -934,7 +934,7 @@
 &usbotg2 {
 	dr_mode = "otg";
 	extcon = <0>, <&typec2_ptn5110>;
-	status = "disabled";
+	status = "okay";
 };
 
 &usdhc1 {
-- 
1.7.9.5

